pkg/controller/certificatesigningrequests/sync_test.go

```diff
@@ -135,6 +144,84 @@ func TestController_Sync(t *testing.T) {
 					Type: certificatesv1.CertificateApproved,
 				})),
 		},
+		"Duration annotation has been provided, but is invalid": {
+			builder: &testpkg.Builder{
+				CertManagerObjects: []runtime.Object{
+					gen.ClusterIssuer("foo-issuer",
+						gen.SetIssuerSelfSigned(cmapi.SelfSignedIssuer{})),
+				},
+				ExpectedEvents: []string{
+					"Warning ErrorParseDuration Failed to parse requested duration: failed to parse requested duration on annotation \"experimental.cert-manager.io/request-duration\": time: invalid duration \"foo\"",
```

Comment:
Any reason this is using double quotes for the string and `\"` instead of using backticks for the string and quotes for the embedded content as in 172?